### ISSUE TITLE
change to JOSS workflow for final acceptance

### ISSUE THREAD

**Issue by danielskatz** [openjournals/joss#762]
Currently, an editor tells whedon to accept, and whedon tries to turn the work over to the AEiC, with
> 👋 @openjournals/joss-eics, this paper is ready to be accepted and published.
>
> Check final proof 👉 openjournals/joss-papers#...
>
> If the paper PDF and Crossref deposit XML look good in openjournals/joss-papers#1523, then you can now move forward with accepting the submission by compiling again with the flag deposit=true e.g.
>
> ```@whedon accept deposit=true```

See https://github.com/openjournals/joss-reviews/issues/2168#issuecomment-651877758

But sometimes, the editor is not really done, and is just checking how things look.

I think we need to separate these two functions - pre-acceptance checks and editor handoff.

I would like @whedon accept to return something like 
> Check final proof 👉 openjournals/joss-papers#...
>
>Editor: If the paper PDF and Crossref deposit XML look good in openjournals/joss-papers#1523, then you can, please let the AEiCs know by
> 
> ```@whedon accept editor-signoff=true```
>
>
>AeIC: If the editor has signed off and the paper PDF and Crossref deposit XML look good in openjournals/joss-papers#..., then you can now move forward with accepting the submission by compiling again with the flag deposit=true e.g.
> 
> ```@whedon accept deposit=true```

And we change the action where whedon adds the "recommended-accept" label to happen after the editor does a ```@whedon accept editor-signoff=true```

**Comment by danielskatz** [comment]
Sorry for the false assignment, but I would appreciate thoughts from @kyleniemeyer @kthyng @labarba and @Kevin-Mattheus-Moerman, and any interested editors

**Comment by arfon** [comment]
👍 good idea. This should be relatively easy to implement.

**Comment by kthyng** [comment]
Makes sense to me. I think this was changed recently to ping EICs, but is fine with me to break it into pieces. Does this need to updated in some docs?

**Comment by kyleniemeyer** [comment]
I guess I'm confused as to why this is needed—what would an editor need to check beyond building the PDF and having whedon check the references?

**Comment by kyleniemeyer** [comment]
As far as I can tell, the only thing otherwise included in the `accept` stage is building the XML—is everyone actually going through that carefully? Because I am not...

**Comment by danielskatz** [comment]
Some editors use this step to check the pdf - the problem is if they are doing this, but the AEiC is proceeding to accept, because they don't know to wait.

The other option would be to better train all editors to not accept until they are satisfied with the PDF via generate

**Comment by danielskatz** [comment]
> As far as I can tell, the only thing otherwise included in the accept stage is building the XML—is everyone actually going through that carefully? Because I am not...

I still skim it just to make sure it looks ok, mostly authors, title, references

**Comment by danielskatz** [comment]
> As far as I can tell, the only thing otherwise included in the accept stage is building the XML—is everyone actually going through that carefully? Because I am not...

I still skim it just to make sure it looks ok, mostly authors, title, references

**Comment by labarba** [comment]
> Some editors use this step to check the pdf - the problem is if they are doing this, but the AEiC is proceeding to accept, because they don't know to wait.

WHAT? 

AEiCs are proceeding to accept _without going through_ checks on the version, archive, references, pdf? Heck, I even read the paper and provide final editorial suggestion.

If AEiCs are publishing _pro forma_, we need to take this more seriously.

**Comment by danielskatz** [comment]
No, AEiCs are not proceeding to accept without checks. Editors (not AEiCs) are using `@whedon accept` as a way to proof everything.  This makes it unclear to AEiC what the status is after an editor does this.  It's an issue where the handoff is unclear.

**Comment by kyleniemeyer** [comment]
If editors are using `whedon accept` just to check things, then that is a misuse of the command, and we should correct that.

@labarba I do think that we are all doing those checks—or at least I am. Occasionally it doesn't seem like the regular editors are doing all of them, however.

**Comment by labarba** [comment]
Then, that calls for a reminder to handling editors of their tasks and the workflow. They should be using `@whedon check references` and `@whedon generate pdf`, then reading the paper on final time, checking that the version is the right one in the repo and it matches what they did with `@whedon add <tag> as version`, and checking that the Zenodo archive link is good and the title and author list match the paper.

It's not uncommon for people to need a reminder once in a while of their tasks and workflow. The time has come.

**Comment by dfm** [comment]
One thing I'd add here: I've ended up asking authors to check over the proofs after executing `@whedon accept` because of the order of steps in the docs: https://joss.readthedocs.io/en/latest/editing.html#after-reviewers-recommend-acceptance I understand that that's not how it's supposed to go, but perhaps moving the "It’s also a good idea to ask the authors to check the proof..." section to a bullet point in the workflow rather than at the end would help?

**Comment by kthyng** [comment]
Yeah, would make sense to have editors have authors check their paper after the editor checks it, in the bullet list.

**Comment by danielskatz** [comment]
I'm going to close this, as I don't think it's an issue with our current workflow and editorialbot commands. Anyone who disagrees can reopen it and ideally, say what they think the issue still is
